email/1.5/modules/ROOT/pages/email-troubleshooting.adoc

```diff
@@ -0,0 +1,238 @@
+. Navigate to the *Console* view to read the email messages in the logger:
++
+[source,plain-text]
+----
+DEBUG: JavaMail version 1.6.3
+DEBUG: successfully loaded resource: /META-INF/javamail.default.address.map
+DEBUG: getProvider() returning javax.mail.Provider[STORE,imap,com.sun.mail.imap.IMAPStore,Oracle]
+DEBUG IMAP: mail.imap.fetchsize: 16384
+DEBUG IMAP: mail.imap.ignorebodystructuresize: false
+DEBUG IMAP: trying to connect to host "******", port 993, isSSL true
+* OK The Microsoft Exchange IMAP4 service is ready. [******]
+A0 CAPABILITY
+* CAPABILITY IMAP4 IMAP4rev1 AUTH=PLAIN AUTH=XOAUTH2 SASL-IR UIDPLUS MOVE ID UNSELECT CHILDREN IDLE NAMESPACE LITERAL+
+A0 OK CAPABILITY completed.
+...
+----
+
+If you run the app in Anypoint Studio, the logging prints the debug lines in the console view. +
+If you run the app in a standalone Mule, the logging prints the debug lines in the log file.
+
+
+== Enable Email Connector Debug Logging
+
+To begin troubleshooting Email Connector, enable debug logging to see the exact error messages:
+
+. Access Anypoint Studio and navigate to the *Package Explorer* view.
+. Open your application's project name.
+. Open the `src/main/resources` path folder.
+. Open the `log4j2.xml` file inside the folder.
+. If the following line is already in the `log4j2.xml` file, uncomment it to enable it; otherwise, add it:
++
+[source,xml,linenums]
+----
+<AsyncLogger name="org.mule.extension.email" level="DEBUG" />
+----
++
+[start=6]
+. Save your changes.
+. Click the project name in *Package Explorer* and then click *Run* > *Run As* > *Mule Application*.
+. Navigate to the *Console* view to read the Email Connector's messages in the logger:
++
+[source,plain-text]
+----
+DEBUG 2021-04-26 18:51:08,536 [_pollingSource_imapattachmentFlow/executor.01] [processor: ; event: ] org.mule.extension.email.internal.mailbox.BaseMailboxPollingSource: Poll will be skipped, since last poll emails are still being processed
+INFO  2021-04-26 18:51:09,166 [[MuleRuntime].uber.04: [imapattachment].imapattachmentFlow.CPU_LITE @111719e0] [processor: imapattachmentFlow/processors/1/processors/2; event: 820f7fe0-a6d9-11eb-a84b-147dda4dba09] org.mule.runtime.core.internal.processor.LoggerMessageProcessor: "" as Binary {base: "64"}
+DEBUG 2021-04-26 18:51:15,360 [_pollingSource_imapattachmentFlow/executor.01] [processor: ; event: ] org.mule.extension.email.internal.mailbox.BaseMailboxPollingSource: Email [172] was not processed.
+....
+----
+
+== Troubleshoot Parsing Text File Attachment as Body Message
+
+When Email Connector receives a message without body and only an attachment file with Content-Type as `text`, the connector publishes the text content as payload body. To receive the text file in the payload attachments list, set the `parsing.text.attachment.as.body` system property to `false`. Refer to the xref:email-attachment.adoc[set parsing text file attachment as body message] documentation for details.
```

Comment:
body > a body
with Content-Type - with the Content-Type `text`,
as a payload body